js/hotgraphicView.js

```diff
@@ -32,6 +32,7 @@ define([
 
         setUpEventListeners: function() {
             this.listenTo(Adapt, 'device:changed', this.reRender);
+            this.listenTo(Adapt, 'device:resize', this.setupTooltips);
```

Comment:
could you do this object-style, please? i.e.
```suggestion
            this.listenTo(Adapt, {
                'device:changed': this.reRender,
                'device:resize': this.setupTooltips
            });
```